srs_data_management/evaluate_model.py

```diff
@@ -6,21 +6,39 @@
 class EvaluateModel:
 
     def __init__(self, config):
-        self._baseline = config['metric']['baseline']
-        self._output_dir = config['output_dir']
-
-    def get_observed_metrics(self, key, level='1*'):
-        process = Popen(['pythona',
-                        '-m', 'asapp.metrics',
-                        '--source', 'comcast_baseline',
-                        '--observed-data', os.path.join('local://srs_data' , key +'_observed.csv'),
-                        '--business-logic', os.path.join(constants.ASAPP_COMCAST_SRS_ROOT , 'business_logic'),
-                        '--metrics', 'cust,acc,prec,recall',
-                        '--blacklist-classes', 'V,_*',
-                        '--taglevel', level
-                        ],
-                       stdout=PIPE,
-                       stderr=PIPE)
+        self._client = config['client']
+        metrics_config = config['metrics']
+        self._baseline = metrics_config['metric']['baseline']
+        self._output_dir = metrics_config['output_dir']
+
+    def get_observed_metrics(self, key, level='1*', blacklist=None, whitelist=None):
```

Comment:
I'd like to move towards calling `blacklist` --> `exclude_list`, `whitelist` --> `include_list`. Can we do that here?

Reply:
when this gets changed, I think we should change them all at the same time instead of having it being half and half. for the ease of merging this into mleng, it makes sense to me to keep it consistent for now